### PR TITLE
fixed mardown formatting in NetInfo code to allow proper JS Docs 

### PR DESCRIPTION
**Pull request by bestander**
Current docs look bad https://facebook.github.io/react-native/docs/netinfo.html#content

### CODE REVIEW

**Comment by mkonicek** [comment]
Thanks a lot!

You can check how it looks locally using

    git clone ... 
    cd react-native/website
    npm install
    npm start

File: Libraries/Network/NetInfo.js [hunk 1]
```diff
@@ -124,6 +125,7 @@ var _isConnectedSubscriptions = new Map();
  * - `WIFI` - The WIFI data connection.
  * - `WIMAX` -  The WiMAX data connection.
  * - `UNKNOWN` - Unknown data connection.
```
Comment (mkonicek) [inline Libraries/Network/NetInfo.js]: Not sure why this list doesn't render as a list.

There's a list here:
https://facebook.github.io/react-native/docs/progressbarandroid.html#content
(source: https://github.com/facebook/react-native/blob/master/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js)

Comment (bestander) [inline Libraries/Network/NetInfo.js]: Yep, it needs a space before the list

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**